lms/templates/verify_student/make_payment_step.underscore

```diff
@@ -31,10 +31,13 @@
       <% if ( _.some( requirements, function( isVisible ) { return isVisible; } ) ) { %>
       <p class="instruction-info">
         <% if ( verificationDeadline && isActive) { %>
-          <%- _.sprintf(
-            gettext( "You can pay now even if you don't have the following items available, but you will need to have these by %(date)s to qualify to earn a Verified Certificate." ),
-            { date: verificationDeadline }
-          ) %>
+          <p
+            class="localized-datetime"
+            data-datetime="<%-verificationDeadline%>"
+            data-string="<%-gettext( "You can pay now even if you don't have the following items available, but you will need to have these by {date} to qualify to earn a Verified Certificate." )%>"
+            data-timezone="<%-userTimezone%>"
+            data-language="<%-userLanguage%>"
+          >
```

Comment:
Screencap of this as it currently stands:

<img width="1231" alt="screen shot 2016-11-17 at 4 32 53 pm" src="https://cloud.githubusercontent.com/assets/7373924/20408458/8e91ed60-ace3-11e6-87b6-d24c46ec3611.png">